packages/experimental/Switch/src/SwitchTokens.win32.ts

```diff
@@ -0,0 +1,99 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { SwitchTokens } from './Switch.types';
+
+export const defaultSwitchTokens: TokenSettings<SwitchTokens, Theme> = (t: Theme) => ({
+  borderWidth: 1,
+  borderRadius: 10,
+  minHeight: 28,
+  minWidth: 40,
+  thumbSize: 14,
+  thumbRadius: 17,
+  trackHeight: 20,
+  trackWidth: 40,
+  focusBorderRadius: 4,
+  focusBorderWidth: 2,
+  focusStrokeColor: t.colors.transparentBackground,
+  padding: 2,
+  color: t.colors.neutralForeground2,
+  trackMarginTop: 2,
+  trackMarginBottom: 2,
+  trackMarginLeft: 2,
+  trackMarginRight: 2,
+  thumbMargin: 2,
+
+  beforeContent: {
+    trackMarginLeft: 8,
+  },
+
+  afterContent: {
+    trackMarginRight: 8,
+  },
+
+  before: {
+    flexDirection: 'row',
+    toggleContainerFlexDirection: 'row',
+  },
+
+  above: {
+    flexDirection: 'column',
+    toggleContainerFlexDirection: 'row',
+  },
+
+  after: {
+    flexDirection: 'row-reverse',
+    toggleContainerFlexDirection: 'row-reverse',
+  },
+
+  toggleOn: {
+    trackColor: t.colors.compoundBrandBackground1,
+    thumbColor: isHighContrast(t) ? t.colors.neutralForegroundOnBrandHover : t.colors.neutralForegroundInverted,
+    borderColor: t.colors.compoundBrandBackground1,
+    justifyContent: 'flex-end',
+    hovered: {
+      trackColor: isHighContrast(t) ? t.colors.neutralForegroundOnBrandHover : t.colors.compoundBrandBackground1Hover,
+      thumbColor: isHighContrast(t) ? t.colors.compoundBrandBackground1Hover : t.colors.neutralForegroundInvertedLink,
+      borderColor: t.colors.compoundBrandBackground1Hover,
+    },
+    pressed: {
+      trackColor: t.colors.compoundBrandBackground1Pressed,
+      thumbColor: isHighContrast(t) ? t.colors.neutralForegroundOnBrandHoverPressed : t.colors.neutralForegroundInvertedLink,
+      borderColor: t.colors.compoundBrandBackground1Pressed,
+    },
+    disabled: {
+      trackColor: isHighContrast ? t.colors.neutralForegroundDisabled : t.colors.neutralBackgroundDisabled,
```

Comment:
Should this be isHighContrast(t) as well?

Reply:
thanks for catching that!